src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -115,6 +119,16 @@
      */
     private int exceptionTimer = 1;
 
+    /**
+     * Check to see if the worker wants to stand still waiting for the request to be fullfilled.
+     */
+    private boolean waitForRequest = true;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "waitForRequest" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)